controllers/managedocs_controller.go

```diff
@@ -39,25 +40,25 @@ import (
 )
 
 const (
-	storageClusterName         = "ocs-storagecluster"
-	addonSecretName     string = "addon-ocs-converged-parameters"
-	storageClassSizeKey string = "size"
+	storageClusterName   = "ocs-storagecluster"
+	addonParamSecretName = "addon-ocs-converged-parameters"
```

Comment:
The actual add-on name should not be a const in the code as this codebase will back all the add-ons for our offering. Even now this const will conflict when deploying with the `ocs-converged-dev` add-on

Reply:
Done